clients/python/test/e2e/ci/e2e/e2e/_data_classes.py

```diff
@@ -0,0 +1,120 @@
+import configparser
+from pathlib import Path
+from typing import Dict
+
+import osparc
+from packaging.version import InvalidVersion, Version
+from pydantic import BaseModel, field_validator
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+# Holds classes for passing data around between scripts.
+
+
+class ServerConfig(BaseSettings):
+    """Holds data about server configuration"""
+
+    host: str
```

Comment:
TIP: usually env vars are CAPITALIZED and we experienced that it more maintainable to map one to one. For that reason in osparc-simcore all the fields of our BaseSettings subclasses are capitalized

If this would be in osparc-simcore, it would be as

```python

class ServerSettings(BaseSettings):
    HOST: IDStr
    KEY: IDStr
    SECRET: SecretStr
```

Reply:
OK, valid point. I have an aversion to capitalization 🙂, so I think I will try to keep it like this until I fall over my own legs and will have to change it. But I see your point...